macros.d/macros.jpackage

```diff
@@ -90,7 +90,10 @@
 #==============================================================================
 # ---- default Java commands
 
-%ant            JAVA_HOME=%{java_home} ant
+%ant \
+echo "[WARNING] Deprecated %%ant macro is used." \
```

Comment:
This wouldn't work without semicolon before backslash. But we agreed that %ant should not be deprecated, so please revert it.